validation.py

```diff
@@ -227,8 +240,8 @@ def run(self):
     parser.add_argument('--branch', '-b', type=str, default=default_branch,
                         help=('Remote repository branch to clone validations '
                               'from. Defaults to %s' % default_branch))
-    parser.add_argument('--regenerate', action='store_true',
-                        help='Re-generate the configuration file')
```

Comment:
At line 222, we still make reference to the --regenerate argument.

Reply:
Done